ariadne/cardano/src/Ariadne/Config/CLI.hs

```diff
@@ -48,6 +50,7 @@ data CLI_AriadneConfig = CLI_AriadneConfig
     , cli_acWallet :: !CLI_WalletConfig
     , cli_acUpdate :: !CLI_UpdateConfig
     , cli_acHistory :: !CLI_HistoryConfig
+    , cli_acLogging :: !CLI_LoggingConfig
```

Comment:
Why do we have this copy-paste?

Reply:
(I mean most of this file)

Reply:
Because nobody knows how to do better.

Reply:
I don't understand the problem solved by this copy-paste.

Reply:
Because we have a configuration stored in file(s) and we want each field there to be overridable via CLI. For each mandatory field in the configuration we want a corresponding optional option in CLI.

Reply:
Hmm, ok. It was hard to notice any difference between the types.